server/message_hooks.go

```diff
@@ -69,106 +74,127 @@ func (p *Plugin) MessageHasBeenPosted(_ *plugin.Context, post *model.Post) {
 				p.API.LogWarn("Unable to handle message sent", "error", err.Error())
 			}
 		}
-		return
+		return nil
 	}
 
 	user, _ := p.API.GetUser(post.UserId)
 
 	_, err = p.Send(link.MSTeamsTeam, link.MSTeamsChannel, user, post)
 	if err != nil {
 		p.API.LogWarn("Unable to handle message sent", "error", err.Error())
+		return err
 	}
+	return nil
 }
 
 func (p *Plugin) ReactionHasBeenAdded(c *plugin.Context, reaction *model.Reaction) {
+	p.GetMetrics().ObserveMessageHooksEvent("reaction_added")
```

Comment:
See above re: where to measure this. (I won't comment everywhere this applies.)

Reply:
Done